tensorflow/lite/delegates/gpu/common/tasks/relu.cc

```diff
@@ -28,14 +28,21 @@ ElementwiseDescriptor CreateReLU(const ReLUAttributes& attr,
   ElementwiseDescriptor result;
   std::string min_func;
   if (attr.alpha != 0.0f) {
-    min_func = "min(in_value * args.alpha, INIT_FLT(0.0f))";
```

Comment:
I believe this is dead code. See tensorflow/lite/core/c/builtin_op_data.h. ReluN1To1 does not even have the attribute alpha (only LeakyRelu does).  Therefore since TfLite does not support this, this will never be reached. Likewise can get rid of changes for alpha != 0 in gl/kernels/relu.cc & tests which have alpha != 0.

Reply:
@grantjensen 
I made this change, the impact is that if ReLUAttributes.alpha != 0, then ReLUAttributes.activation_min is ignored.
It makes the unit tests a little confusing.